public/app/themes/justice/inc/search.php

```diff
@@ -265,20 +265,22 @@ public function relevanssiParentFilter(array $hits): array
         if (!isset($wp_query->query_vars['parent'])) {
             // No parent parameter set, do nothing.
             return $hits;
+        } else {
+            $parent_id = $wp_query->query_vars['parent'];
         }
 
         $acc = [];
         foreach ($hits[0] as $hit) {
             // Loop through all the posts found.
-            if ($hit->ID === $wp_query->query_vars['parent']) {
+            if ($hit->ID === $parent_id) {
```

Comment:
`switch` is faster than multiple if, else

```php
switch (true) {
    case ($hit->ID === $parent_id):
        break;
    case ($hit->post_parent === $parent_id): 
        break;
    case ($hit->post_parent > 0):
        break;
}
```

Reply:
Thanks @wilson1000, I didn't actually write this it was already there. I think the formatter just adjusted a few things, happy to refine it though :) 

I'm not sure that's always true, PHP Bench reckons they're about the same https://phpbench.com/
![Screenshot 2024-06-20 at 09 33 41](https://github.com/ministryofjustice/justice-gov-uk/assets/24455584/5bde0732-1f04-4ccc-8203-d1a4ecd6f706)

In which case I suppose it's down to our coding standards/readability.

Reply:
Yes, I'd challenge that. I did a benchmark a while ago and the difference was quite a bit. Although, there's nothing to say the speed of `multi if else` hasn't improved with age 😄 - thanks for sharing

Reply:
So here's a funny thing... I clicked the phpbench link and my browser froze lol

Reply:
Haha, maybe PHP Bench isn't the most reliable then... 😆